docs/src/guide/installation.md

```diff
@@ -26,29 +26,23 @@ That's it. With this you are all set to
 run the code in the [Tutorial](@ref) or the
 [`examples` directory](https://dftk.org/tree/master/examples).
 
-## Recommended optional python dependencies
-While not strictly speaking required to use DFTK, a large number of convenience
-features (such as reading standard [Input and output formats](@ref))
-are based on ASE, the [atomistic simulation environment](https://wiki.fysik.dtu.dk/ase/index.html).
-The installation of the ASE python package is therefore strongly recommended.
-To do so first find out which version of python is used by the
-[PyCall.jl](https://github.com/JuliaPy/PyCall.jl) package.
-You can do this for example with the Julia commands
+## Recommended optional packages
+While not strictly speaking required to use DFTK it is usually convenient to install
+a couple of standard packages from the [AtomsBase](https://github.com/JuliaMolSim/AtomsBase.jl)
+ecosystem, such as [AtomsIO.jl](https://github.com/mfherbst/AtomsIO.jl)
+or [ASEconvert.jl](https://github.com/mfherbst/ASEconvert.jl).
+You can install these two packages using
```

Comment:
Fine but won't the user still need to install python ase? This is just shifting the problem to another package, right?

Reply:
No as ASEconvert does this automatically.

Reply:
OK but automatically in python means "it fails only 50% of the time", no?

Reply:
no this is pretty reliable and independent of your system python (makes a custom conda environment and such ... try it, it's pretty cool). Has just one issue: You cannot use PyCall-based packages and PythonCall-based packages at the same time.

Reply:
That means no DFTK+PyPlot? That's a pretty huge caveat! I don't think we should do it until that's fixed... What's the issue number?

Reply:
You can use [PythonPlot](https://github.com/stevengj/PythonPlot.jl) instead. Also it's not an issue more a fundamental thing. The caveat is down to race conditions due to the way the libpython and down-stream dependencies are dynamically loaded into the Julia process. If you want to avoid that you can always opt-out of the automatic python installation stuff in PythonCall and manage everything yourself (using PyCall and conda packages as before). Point is just that novice users don't need to bother with this.

Reply:
Ah turns out this is only an issue if the python interpreters are actually different: https://cjdoris.github.io/PythonCall.jl/stable/pycall/

Reply:
I mean, novice users are just going to install this, then pyplot, then see crashes. Probably we should add a note somewhere, maybe including the error message so that people can find it when googling.

Reply:
I should probably add some details in the docs about this.